### ISSUE TITLE
Turing Pi

### ISSUE THREAD

**Issue by qskwood** [geerlingguy/raspberry-pi-dramble#188]
You should mention Turing Pi in the hardware section with a link to their website. I had a hard time finding it.

https://turingpi.com/

**Comment by geerlingguy** [comment]
Good idea; I have a completely separate project that installs K3s, Drupal, Wordpress, and some other tools on the Turing Pi here: https://github.com/geerlingguy/turing-pi-cluster
